odr-dabmux.service

```diff
@@ -6,6 +6,9 @@ After=network-online.target sound.target
 
 [Service]
 Type=simple
+ExecStartPre=-/bin/mkdir -p /var/tmp/dabmux
+ExecStartPre=-/bin/mkfifo --mode=644 /var/tmp/dabmux/odr-dabmux-local-01.fifo
+ExecStartPre=-/bin/chown odr-dabmux:odr-dabmux /var/tmp/dabmux/odr-dabmux-local-01.fifo
 ExecStart=/usr/bin/odr-dabmux /etc/odr-dabmux/dab.mux
 User=odr-dabmux
 Group=odr-dabmux
```

Comment:
> I feel writing all of that is rather overkill given our current use case (local feedback) and our small user base (us). I'd be happy to to do (and properly test) that down the line when this has gotten more use.

If you fell that this is overkill, why then even bother with the fifo creation at all? You could simply apply the fix from [247638771](https://github.com/Opendigitalradio/ODR-DabMux/issues/18#issuecomment-247638771) and let the odr-dabmux create the fifo for you.